Modulefile

```diff
@@ -1,5 +1,5 @@
 name    'ghoneycutt-dnsclient'
-version '3.1.1'
+version '3.2.0'
```

Comment:
I'm not sure how that change, and the additions of 7 to all the rhel based operating systems happened.  I thought I forked from master then edited from there, but it looks like I grabbed something unintentionally (I'm still a novice at git, so I hope you'll excuse that.)